.github/workflows/gh-pages.yml

```diff
@@ -7,10 +7,13 @@ on:
 jobs:
   deploy:
     runs-on: ubuntu-latest
+    env:
+      TX_TOKEN: ${{ secrets.TX_TOKEN }}
```

Comment:
Can this be set for the necessary steps only? (minimize potential attack surface)